src/cmds/upgrade.js

```diff
@@ -0,0 +1,68 @@
+const { execSync } = require('child_process')
+const { default: fetch } = require('node-fetch')
+const prompts = require('prompts')
+const semverGte = require('semver/functions/gte')
+ 
+const configStore = require('../helpers/configStore.js')
+const checkCurrentVersion = require('./version').getModuleVersion
+
+async function checkLatestVersion() {
+  const response = await fetch('https://registry.npmjs.org/@livestorm/cli')
+  const json = await response.json()
+  return json['dist-tags']['latest']
+}
+
+async function checkCandidateForUpgrade() {
+  try {
+    const currentDate = new Date().toISOString().split('T')[0]
+    const latestUpgradeDate = configStore.get('latestUpgradeDate')
+    if (currentDate === latestUpgradeDate) return false
+
+    configStore.set('latestUpgradeDate', currentDate)
+
+    const currentVersion = checkCurrentVersion()
+    const latestVersion = await checkLatestVersion()
+    if (semverGte(currentVersion, latestVersion)) return false
+
+    return true
+  }
+  catch (error) {
+    return false
+  }
+}
+
+async function promptUpgrade() {
+  prompts({
+    type: 'text',
+    name: 'upgrade',
+    message: "We noticed your CLI isn't up to date, do you want to upgrade? (yes/no)",
+    validate: value => {
+      return (value !== 'no' || value !== 'yes')
+    }
+  }).then(answer => {
+    if (answer.upgrade === 'yes') return true
+
+    return false
+  })
+}
```

Comment:
Your method does not return anything, this is what you should be doing : 
```javascript
async function promptUpgrade() {
  const answer = await prompts({
    type: 'text',
    name: 'upgrade',
    message: "We noticed your CLI isn't up to date, do you want to upgrade? (yes/no)",
    validate: value => {
      return (value !== 'no' || value !== 'yes')
    }
  })
  
  if (answer.upgrade === 'yes') return true
  return false
}
```

This will allow your code below : 
``` 
  const yes = await promptUpgrade()
```
to work